ParisBNF/abb/BNFabb157.xml

```diff
@@ -0,0 +1,488 @@
+                              <desc type="Correction">Minor corrections on these and other folia, mostly omitted passages added interlineally or in the margins, also 
+                                by means of <foreign xml:lang="am">tamallas</foreign>-signs; rarely passages written over erasure, in smaller script.</desc>
+                           </item>
+
+                           <item xml:id="e4">
+                             <locus target="#221r"/>
+                              <desc type="StampExlibris">Stamp of the Library.</desc>
+                           </item>
+                        </list>
+
+                  </additions>
+                  </physDesc>
+                          </msPart>
+
+
+
+
+               <msPart xml:id="p2">
+                  <msIdentifier/>
+                  <msContents>
+                     <summary/>
+
+                     <msItem xml:id="p2_i1">
+                        <locus from="25ra" to="35v"/>
+                       <title type="incomplete" ref="LIT2477Penta">Excerpts from various books of the Pentateuch, starting with Lev 1, ending
+                       with Ex 2</title>
+                      <textLang mainLang="gez"/>
+                       <note>The rubricated lines are hardly readable on the available images.</note>
+                     </msItem>
+                  </msContents>
+
+                  <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                           <extent>
+                              <measure unit="leaf" quantity="11">11</measure>
+                              <locus from="25r" to="35v"/>
+                           </extent>
+                        </supportDesc>
+
+                        <layoutDesc>
+                           <layout columns="2" writtenLines="37 40">
+                             <note>The number of written lines varies greatly on other folia.</note>
+                      <ab type="pricking">Pricking is hardly visible.</ab>
+                             <ab type="ruling">Ruling is hardly visible.</ab>
+                              <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                           </layout>
+                        </layoutDesc>
+                     </objectDesc>
+
+                     <handDesc>
+                        <handNote xml:id="h2" script="Ethiopic" corresp="p1_i2">
+                          <locus from="25r" to="35v"/>
+                         <seg type="script">Second half of the 19th-century script.</seg>
+                          <seg type="ink">Black; red.</seg>
+                           <desc>Careless, irregular; by a poor scribe.
```

Comment:
Does "poor" refer to material wealth or to intelectual qualities? Perhaps it might  be better to describe it as "Careless, irregular and humble".

Reply:
it is a kind of estimation referring to the level of the scribe that I personally use (well-trained, mediocre, poor = poorly trained, followed by crude hands).

Reply:
OK. I will use these categories in describing hands in future as well, but I will prefer "poorly trained" to avoid misunderstanding.